File: Flutter/lib/model/voiceroom_seat_event.dart
// Copyright (c) 2022 NetEase, Inc. All rights reserved.
// Use of this source code is governed by a MIT license that can be
// found in the LICENSE file.

class VoiceRoomSeatEvent {
  final String user;

  final int index;

  final int reason;

  VoiceRoomSeatEvent(this.user, this.index, this.reason);

  @override
  String toString() {
    return 'VoiceRoomSeatEvent{user: $user, index: $index, reason: $reason}';
  }
}

File: Android/voiceroomkit-ui-base/src/main/java/com/netease/yunxin/kit/voiceroomkit/ui/base/model/VoiceRoomSeatEvent.java
// Copyright (c) 2022 NetEase, Inc. All rights reserved.
// Use of this source code is governed by a MIT license that can be
// found in the LICENSE file.

package com.netease.yunxin.kit.voiceroomkit.ui.base.model;

public class VoiceRoomSeatEvent {

  private String user;

  private int index;

  private int reason;

  public VoiceRoomSeatEvent(String user, int index, int reason) {
    this.user = user;
    this.index = index;
    this.reason = reason;
  }

  public String getUser() {
    return user;
  }

  public void setUser(String user) {
    this.user = user;
  }

  public int getReason() {
    return reason;
  }

  public void setReason(int reason) {
    this.reason = reason;
  }

  public int getIndex() {
    return index;
  }

  public void setIndex(int index) {
    this.index = index;
  }
}
